### PR TITLE
Add ViT JAX notebook.

### PR DESCRIPTION
**Pull request by lavraicse**
<br>
Adding a notebook for JAX based ViT-B16 model finetuning and prediction.
<br><br><br>

- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [x] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/community/model_garden/model_garden_jax_vision_transformer.ipynb [hunk 1]
```diff
@@ -0,0 +1,855 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_jax_vision_transformer.ipynb:1]: <p>Explain what these functions do</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1997/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_jax_vision_transformer.ipynb:1]: <p>TensorFlow datasets -&gt; TensorFlow Datasets</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1997/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_jax_vision_transformer.ipynb:1]: <p>no future tense: drop will</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1997/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_jax_vision_transformer.ipynb:1]: <p>drop "will"</p><p>TF-Saved model -&gt; TF SavedModel format</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1997/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_jax_vision_transformer.ipynb:1]: <p>same as prev cell</p><p> </p><p>Explain steps below, upload to MR, deploy model, prepare test instances, make prediction.</p><p> </p><p>Perhaps add a text cell for each code cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1997/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Comment (andrewferlitsch) [reply notebooks/community/model_garden/model_garden_jax_vision_transformer.ipynb:1]: <p>in cell 'Define common functions'</p>

Comment (andrewferlitsch) [reply notebooks/community/model_garden/model_garden_jax_vision_transformer.ipynb:1]: <p>in cell Finetune with JAX</p>

Comment (andrewferlitsch) [reply notebooks/community/model_garden/model_garden_jax_vision_transformer.ipynb:1]: <p>in cell Convert JAX ...</p>

Comment (andrewferlitsch) [reply notebooks/community/model_garden/model_garden_jax_vision_transformer.ipynb:1]: <p>In cell Run online prediction</p>

Comment (lavraicse) [reply notebooks/community/model_garden/model_garden_jax_vision_transformer.ipynb:1]: Done.

Comment (lavraicse) [reply notebooks/community/model_garden/model_garden_jax_vision_transformer.ipynb:1]: Done.

Comment (lavraicse) [reply notebooks/community/model_garden/model_garden_jax_vision_transformer.ipynb:1]: Done.

Comment (lavraicse) [reply notebooks/community/model_garden/model_garden_jax_vision_transformer.ipynb:1]: Done.

Comment (lavraicse) [reply notebooks/community/model_garden/model_garden_jax_vision_transformer.ipynb:1]: Done.

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**